spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/deployer/BackingServicesProvisionService.java

```diff
@@ -28,4 +29,15 @@
 
 	Flux<String> deleteServiceInstance(List<BackingService> backingServices);
 
+	/**
+	 * Create service keys
+	 * @return A flux of credentials associated with each created service key
```

Comment:
Please, delete comment to be consistent with the other methods.

Reply:
As the content of the returned value is assigned deep in CloudFoundryAppDeployer, and felt not that intuitive obvious to newcomers, I rather added javadoc on all methods of the class as to still preserve class consistency. Please let me know if you have a better suggestion on how to remove ambiguity to code readers on the intent and content of the returned type.